src/easydmp/dmpt/models.py

```diff
@@ -2210,13 +2209,12 @@ def validate_choice(self, data):
         return False
 
 
-class DoSectionQuestion(NoCheckMixin, Question):
+class DoSectionQuestion(BooleanQuestion):
```

Comment:
No need to change INPUT_TYPES this way...

Reply:
On the other hand, I suspect if this works, we don't need a separate proxy Question-class at all, just an ordinary BooleanQuestion, If it otherwise is identical to BooleanQuestion. If it isn't it (extra methods, say) we will need a new input type regardless.